### PR TITLE
Merge gitattributes

### PR DESCRIPTION
**Pull request by dgorodnichy**

### CODE REVIEW

**Comment by dgorodnichy** [comment]
@pftg , You can check taht gitattributes were merged with this command:
```
╰─ $ git check-attr -a db/schema.rb
db/schema.rb: diff: ruby
db/schema.rb: merge: ours
db/schema.rb: linguist-generated: set
```

File: template.rb [hunk 1]
```diff
@@ -104,7 +104,19 @@ def add_template_repository_to_source_path
...
+  *.gemspec diff=ruby
+  *.rake    diff=ruby
+  *.rb      diff=ruby
+  *.js      diff=javascript
+
+  db/schema.rb merge=ours diff=ruby
+  yarn.lock merge=ours
+  Gemfile.lock merge=ours linguist-generated
+
+  vendor/* linguist-vendored
```
Comment (pftg) [inline template.rb:117]: You can remove this, it’s already in the target file

**Review by pftg — CHANGES_REQUESTED** [review]
Please check

File: template.rb (hunk 1 above)
Comment (dgorodnichy) [reply template.rb]: Done

**Review by pftg — APPROVED** [review]
LGTM

File: template.rb [hunk 2]
```diff
@@ -104,7 +104,17 @@ def add_template_repository_to_source_path
...
+  *.gemspec diff=ruby
+  *.rake    diff=ruby
+  *.rb      diff=ruby
+  *.js      diff=javascript
+
+  db/schema.rb merge=ours diff=ruby
+  yarn.lock merge=ours
+  Gemfile.lock merge=ours linguist-generated
+CODE
+insert_into_file '.gitattributes', gitattributes
```
Comment (pftg) [inline template.rb:117]: but what to do for rails 6.0? there are no such file/

**Review by pftg — CHANGES_REQUESTED** [review]
Please check for rails "< 6.1" there are no such files

File: template.rb [hunk 3]
```diff
@@ -104,7 +104,17 @@ def add_template_repository_to_source_path
 
 say "=> Copying git configuration"
 directory "lib/install/.github", ".github"
-copy_file "lib/install/.gitattributes", ".gitattributes"
+gitattributes = <<-CODE
```
Comment (pftg) [inline template.rb:107]: Please use `~` instead of `-` and rename `CODE` to something meaninful

File: template.rb (hunk 2 above)
Comment (dgorodnichy) [reply template.rb]: Updated

File: template.rb (hunk 3 above)
Comment (dgorodnichy) [reply template.rb]: Updated

**Pull request merged**